### ISSUE TITLE
[BUG] Can't create views with different storage paths

### ISSUE THREAD

**Issue by Zoldex** [paperless-ngx/paperless-ngx#1462]
### Description

After filtering documents with different storage paths, trying to save a new view from menu views -> save as will output the error 
```
Filter rules error occurred while saving this view
The error returned was:
,,, 
```
![immagine](https://user-images.githubusercontent.com/6499525/186914881-dbdadb94-a395-4053-802e-88590a7995cc.png)

### Steps to reproduce

1. Go to Documents
2. Filter documents by storage paths
3. click on menu Views / Save As
4. input any View name
5. click on Save

### Webserver logs

```bash
[2022-08-26 15:32:59,349] [WARNING] [django.request] Bad Request: /api/saved_views/
```

EDIT:
```
{
    "filter_rules": [
        {},
        {
            "rule_type": [
                "\"25\" is not a valid choice."
            ]
        }
    ]
}
``


### Paperless-ngx version

1.8.0

### Host OS

Docker

### Installation method

Docker - official image

### Browser

Any

### Configuration changes

      PAPERLESS_FILENAME_FORMAT: "Casa/{correspondent}/{document_type}/{created_year}/{tag_list}/{created_month}-{created_day} {title}"PAPERLESS_FILENAME_FORMAT_REMOVE_NONE: True

### Other

_No response_

**Comment by shamoon** [comment]
Ah, good catch thanks. I'll look into this

**Comment by qcasey** [comment]
Fixed in #1463
